kover-gradle-plugin/src/main/kotlin/kotlinx/kover/gradle/plugin/dsl/KoverReportConfig.kt

```diff
@@ -0,0 +1,1225 @@
+ *  log {
+ *      onCheck = true
+ *
+ *      filters {
+ *          // ...
+ *      }
+ *      header = null
+ *      format = "<entity> line coverage: <value>%"
+ *      groupBy = GroupingEntityType.APPLICATION
+ *      coverageUnits = MetricType.LINE
+ *      aggregationForGroup = AggregationType.COVERED_PERCENTAGE
+ *  }
+ * ```
+ */
+public interface KoverLogTaskConfig {
+    @Deprecated(
+        message = "It is forbidden to override filters for a specific report, use custom report variants to create reports with a different set of filters. Please refer to migration guide in order to migrate: ${KoverMigrations.MIGRATION_0_7_TO_0_8}",
+        level = DeprecationLevel.ERROR
+    )
+    public fun filters(config: Action<KoverReportFiltersConfig>) {
+        throw KoverDeprecationException("It is forbidden to override filters for the log report, use custom report variants to create reports with a different set of filters. Please refer to migration guide in order to migrate: ${KoverMigrations.MIGRATION_0_7_TO_0_8}")
+    }
+
+    /**
+     * Print coverage when running the `check` task.
+     *
+     * Empty by default.
+     */
+    public val onCheck: Property<Boolean>
+
+    /**
+     * Add a header line to the output before the lines with coverage.
+     *
+     * If value is empty then the header is absent.
+     *
+     * Empty by default.
+     */
+    public val header: Property<String>
+
+    /**
+     * Format of the strings to print coverage for the specified in [groupBy] group.
+     *
+     * The following placeholders can be used:
+     *  - `<value>` - coverage value
+     *  - `<entity>` - name of the entity by which the grouping took place. `application` if [groupBy] is [GroupingEntityType.APPLICATION].
+     *
+     * `"<entity> line coverage: <value>%"` if value is empty.
+     *
+     * Empty by default.
+     */
+    public val format: Property<String>
+
+    /**
+     * Specifies by which entity the code for separate coverage evaluation will be grouped.
+     *
+     *
+     * [GroupingEntityType.APPLICATION] if value is empty.
+     *
+     * Empty by default.
+     */
+    public val groupBy: Property<GroupingEntityType>
+
+    /**
+     * The type of application code division (unit type) whose unit coverage will be considered independently.
+     *
+     * [MetricType.LINE] if value is empty.
+     *
+     * Empty by default.
+     */
+    public val coverageUnits: Property<MetricType>
+
+    /**
+     * Specifies aggregation function that will be calculated over all the units of the same group.
+     *
+     * This function used to calculate the aggregated coverage value, it uses the values of the covered and uncovered units of type [coverageUnits] as arguments.
+     *
+     * Result value will be printed.
+     *
+     * [AggregationType.COVERED_PERCENTAGE] if value is empty.
+     *
+     * Empty by default.
+     */
+    public val aggregationForGroup: Property<AggregationType>
+}
+
+/**
+ * Filters to excludes
```

Comment:
I think something wrong with this wording, but I can't say what 🤔 Perhaps it should be `Filters for report exclusions` or `Filters to exclude`.

Reply:
-> Filters to exclude classes from reports